mitosheet/src/components/taskpanes/FileImport/CSVImportConfigTaskpane.tsx

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Mito
+
+import React from 'react';
+import useSendEditOnClick from '../../../hooks/useSendEditOnClick';
+import MitoAPI from '../../../jupyter/api';
+import { AnalysisData, StepType, UIState } from '../../../types';
+import CSVImportScreen, { CSVImportParams } from '../../import/CSVImportConfigScreen';
+import { ImportState } from './FileImportTaskpane';
+
+interface CSVImportConfigTaskpaneProps {
+    mitoAPI: MitoAPI;
+    analysisData: AnalysisData;
+    setUIState: React.Dispatch<React.SetStateAction<UIState>>;
+
+    fileName: string;
+    filePath: string;
+
+    error: string | undefined;
+    setScreen: React.Dispatch<React.SetStateAction<ImportState>>;
+}
+
+
+export const getDefaultCSVParams = (filePath: string): CSVImportParams => {
+    return {
+        file_names: [filePath],
+        delimeters: [','],
+        encodings: ['default'],
+        error_bad_lines: [true]
+    }
+}
+
+/**
+ * A taskpane that allows a user to configur a CSV taskpane for import
+ * as a new step.
+ */
+function CSVImportConfigTaskpane(props: CSVImportConfigTaskpaneProps): JSX.Element {
+
+    const {params, setParams, loading, edit, editApplied, error} = useSendEditOnClick(
+        () => getDefaultCSVParams(props.filePath),
+        StepType.SimpleImport,
+        props.mitoAPI, props.analysisData,
+        {allowSameParamsToReapplyTwice: true},
+    )
+
+    return (
+        <CSVImportScreen
```

Comment:
Oh can we update the name of this component. It looks like it should be called `CSVImportConfigScreen`